galsim/sed.py

```diff
@@ -413,7 +421,8 @@ def _fast_spec(self):
                     f = self._rest_nm_to_photons(x)
                 else:
                     f = self._rest_nm_to_dimensionless(x)
-                return _LookupTable(x, f, interpolant='linear')
+                interp = self._spec.interpolant if isinstance(self._spec, LookupTable) else 'spline'
```

Comment:
Most of the other default interps look to be 'linear', why 'spline' here?

Reply:
Good catch.  I was originally a little aggressive about trying to get most things to use spline by default, but then I backpedaled on that idea, since it felt a little too API-changing. I just missed this one.